libkbfs/folder_branch_ops.go

```diff
@@ -1824,6 +1825,8 @@ func (fbo *folderBranchOps) isRevisionConflict(err error) bool {
 
 func (fbo *folderBranchOps) finalizeMDWriteLocked(ctx context.Context,
 	lState *lockState, md *RootMetadata, bps *blockPutState, excl EXCL) (err error) {
+	fbo.log.CDebugf(ctx, "finalizeMDWriteLocked: excl=%s", excl)
+	defer func() { fbo.deferLog.CDebugf(ctx, "Done: %v", err) }()
```

Comment:
Sorry for the excessive logs! I added those to help with debugging; will clean them up